brokers/theia/broker.go

```diff
@@ -130,10 +130,10 @@ func (b *Broker) processPlugin(meta model.PluginMeta) error {
 	}
 	if pluginImage == "" {
 		// regular plugin
-		return b.injectTheiaFile(meta, archivePath)
+		return b.injectTheiaFile(meta, archivePath, onlyMetadata)
```

Comment:
The purpose of this method is to inject file. It is better to refactor code instead of just adding boolean parameter to it. And with the remote plugin the problem is the same.

Reply:
I agree that some refactoring might be useful, but my idea was mainly to change the structure of the code as less as possible, knowing that a number of changes are ongoing or planned. I would have let the refactoring phase for the upcoming changes planned in the context of meta.yaml and plugin yaml merge.